features/landrush-ip.feature

```diff
@@ -0,0 +1,63 @@
+Feature: landrush-ip
+  Landrush should pick the desired IP, given multiple network interfaces.
+
+  Scenario Outline: booting a box and picking a specific interface
+    Given a file named "Vagrantfile" with:
+    """
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.hostname = 'my-host.landrush-acceptance-test'
+
+      config.vm.network :private_network, ip: '192.168.50.50'
+      config.vm.network :private_network, ip: '192.168.50.51'
+      config.vm.network :private_network, ip: '192.168.50.52'
+      config.vm.network :private_network, ip: '192.168.50.53'
+      config.vm.network :private_network, ip: '192.168.50.54'
+
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+
+      config.landrush.enabled = true
+      config.landrush.tld = 'landrush-acceptance-test'
+      config.landrush.interface = 'eth3'
```

Comment:
So the explicit include via the _interface_ option overrides the fact that the interface is excluded via the regexp. I guess that might be worth an explicit mention in the README.